setup.py

```diff
@@ -192,7 +193,7 @@ def initialize_options(self) -> None:
 ]
 
 pandas_dependencies = [
-    'pandas==1.1.5',
+    'pandas',
```

Comment:
This is another change that I'm not super comfortable with - why was it pinned at 1.1.5 in the first place, and are we really sure this will work with ANY pandas version? The circleci config currently explicitly tests against pandas 1.1.5 and 1.3.5 (?), I think at minimum we would want to expand that to maybe testing against 1.1.5 and whatever the latest version is.

Reply:
I think you're right about this; having an explicit version is a liability as we go to future versions of python but, until we're there happy to take this out.